main.py

```diff
@@ -28,3 +30,15 @@ def scrape_url(url):
         return response.text
     except requests.exceptions.RequestException as err:
         raise HTTPException(status_code=400, detail=str(err))
+
+
+def extract_info_with_chatgpt(html_content):
+    openai.api_key = 'your-api-key'
+    response = openai.ChatCompletion.create(
+      model="gpt-3.5-turbo",
+      messages=[
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": html_content}
+        ]
```

Comment:
We want to extract a review from the html. Try to extract the name of the reviewer, the review, the rating, the date, and an image (if available)